language/README.rst

```diff
@@ -99,4 +99,4 @@ Next Steps
 -  Read the `Product documentation`_ to learn
    more about the product and see How-to Guides.
 
-.. _Usage documentation: https://googleapis.dev/python/language/latest
+.. _Usage documentation: https://googleapis.dev/python/natural-language/latest
```

Comment:
This new URL does not work (yet?), while the old link does

Reply:
@plamut You are correct that this one is not broken. I've removed this change. Cheers!

Reply:
Thanks!

Reply:
language - they should all match the DNS name. :)